zarr/tests/test_core.py

```diff
@@ -80,15 +125,11 @@ def test_array_init(self):
         assert '' == a.path
         assert a.name is None
         assert a.basename is None
-        assert store is a.store
-        if self.version == 2:
-            assert "8fecb7a17ea1493d9c1430d04437b4f5b0b34985" == a.hexdigest()
-        else:
-            assert "968dccbbfc0139f703ead2fd1d503ad6e44db307" == a.hexdigest()
+        assert a.store == normalize_store_arg(store)
         store.close()
 
         # initialize at path
-        store = self.KVStoreClass(dict())
+        store = self.create_store()
```

Comment:
Kind of feels like we should be in a separate test here. Is anything reused below from above?

Reply:
no, nothing is reused, so it's a great opportunity for just making a separate test